docs/source/index.mdx

```diff
@@ -0,0 +1,148 @@
+---
+title: Getting Started
+description: Get started with Preflight Checks and Support Bundles
+---
+
+import { Link } from "gatsby";
+import { Button } from '@apollo/space-kit/Button';
+import { colors } from 'gatsby-theme-apollo-core';
+
+Welcome.
+The easiest way to get started with the Troubleshoot project is to see it in action through a couple examples. 
+
+## Installation
+Executing Preflight Checks and Support Bundles relies on a client-side utility, packages as a `kubectl` plugin and distributed through the [krew](https://krew.dev/) pacakge manager.
+If you don't already have krew installed, head over to the [krew installation guide](https://krew.sigs.k8s.io/docs/user-guide/setup/install/), follow the steps there and then come back here.
+
+Install the Preflight and Support Bundle plugin using:
+
+```shell
+kubectl krew install preflight
+kubectl krew install support-bundle
+```
+
+Note: This will not install anything to your cluster, it only places a single binary per plugin in your path.
+
+## Examples
+
+Now that you have the plugins installed lets look at a simple example of a Preflight Check and a Support Bundle to get a sense for how they are structured and how to run them.
+
+### Example Preflight check
+
+Preflight Checks can be executed by your customer before installing your application into their Kubernetes cluster. The checks are defined in a YAML file. Here is an example of a simle Preflight Check that could be used to enforce a minimum Kubernetes version on your cluster.
+
+```
+apiVersion: troubleshoot.replicated.com/v1beta1
+kind: Preflight
+metadata:
+  name: kubernetes-version
+  annotations: 
+    title: "Check the version of Kubernetes"
+    description: "Require a minimum of Kubernetes 1.16.0"
+    category: "cluster_info"
+    iconUri: "/category-icons/kubernetes.svg"
+    tags: '["kubernetes"]'
+    contributors: '[{"name": "Marc Campbell", "avatarUri": "https://avatars3.githubu
+spec:
+  collectors: []
+  analyzers:
+    - clusterVersion:
+        outcomes:
+          - fail:
+              when: "< 1.16.0"
+              message: The application requires at Kubernetes 1.16.0 or later, and
+              uri: https://kubernetes.io
+          - warn:
+              when: "< 1.18.0"
+              message: Your cluster meets the minimum version of Kubernetes, but we
+              uri: https://kubernetes.io
+          - pass:
+              message: Your cluster meets the recommended and required versions of K
+```
+
+Ready to try it out? Execute the following command to see the results of this Kubernetes version check Preflight Check.
+
+```
+kubectl preflight https://troubleshoot.run/nCA5lOsuA
```

Comment:
@GraysonNull This line was copied from Figma, but of course isn't actually the right file. Any idea how we can publish a real spec? Also, the spec itself above needs updating.

Reply:
@GraysonNull Oh, the spec may work, but the copy was clipped, so sentences stop mid-word. I don't have the original text. And the URL "works", but downloads a completely different spec. I'd like to have a URL with fixed content. Maybe we should host it at `troublesheet.sh/docs/preflight/example` or something like that.

Reply:
@markpundsack Ah gotcha, yeah that makes sense. The original spec content is from here https://troubleshoot.sh/spec/kubernetes-version

WRT to the fixed content URL: Definitely agree. I can look into how we can host it at that URL. Might be a bit tricky because gatsby wants all of those pages to be markdown so i'm worried it may just 404 if you're trying to access a yaml file like that. Hopefully it just works but I'll look into it